pkg/subscriber/processor/push_sender.go

```diff
@@ -130,7 +131,7 @@ func (p pushSender) handle(msg *puller.Message) {
 }
 
 func (p pushSender) send(featureID, environmentId string) error {
-	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), timeout)
```

Comment:
I think we can make this timeout as configuration, so we can configure it in values.yaml on demand.